.github/workflows/ci.yml

```diff
@@ -2,6 +2,7 @@ name: CI
 
 on:
   pull_request:
+  merge_group:
```

Comment:
Per https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions